test/services/courses/services/scopelist.test.js

```diff
@@ -18,7 +18,7 @@ describe('courses scopelist service', () => {
 		});
 		const archivedCourse = await testObjects.createTestCourse({
 			userIds: [user._id],
-			untilDate: Date.now() - 600000,
+			untilDate: Date.now() - 172800000,
```

Comment:
use a variable instead here and below repetitions

Reply:
done